addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -81,14 +82,27 @@
                         </page>
                     </notebook>
                     <footer>
-                        <button string="Send" name="action_send_mail"
-                                type="object" class="btn-primary o_mail_send" data-hotkey="q"
-                                invisible="subtype_is_log or composition_mode == 'comment' and not composition_batch and scheduled_date"/>
-                        <button string="Log" name="action_send_mail"
-                                type="object" class="btn-primary" data-hotkey="q"
-                                invisible="not subtype_is_log or composition_mode == 'comment' and not composition_batch and scheduled_date"/>
-                        <button string="Schedule" name="action_schedule_message" type="object" class="btn-primary" data-hotkey="q"
-                                invisible="composition_mode != 'comment' or composition_batch or not scheduled_date"/>
+                        <field name="all_partners_have_email" invisible="1"/>
+                        <div invisible="not all_partners_have_email">
```

Comment:
Don't understand difference between those two blocks ^^ anyway not sure what we want ?

Reply:
Normally it is possible, even if a bit tricky, already did it in qweb at least. Worth checking as it makes the view quite hard to understand, and also breaks customized views (and I bet they are quite a lot of them for something that is the mail composer).